### ISSUE TITLE
Feature: exclude certain apps from equalizer

### ISSUE THREAD

**Issue by Miosame** [bitgapp/eqMac#745]
- [x] I have checked for a similar issue and sure it hasn't been reported before.

**Description:**
It would be nice if you could exclude applications from the equalizer

**User flow & interface:**
One could possibly re-use the app-mixer interface, with a tiny icon at the bottom of the volume sliders or next to the icon itself, which when clicked adds it to a block-list.

I'm not certain this is even possible, but for the sake of feature-request, I'll assume eqMac is a per-stream equalizer rather than using some secret macOS API that presents "all" streams.

Thanks!

**Comment by superjeng1** [comment]
Hi @Miosame, I am not sure what you mean by "blocking". Do you mean muting certain apps?

**Comment by Miosame** [comment]
No, I mean excluding it from filters being applied to it, e.g. I have remote software with audio-passthrough where the target also has a bass filter, now both get applied together and it becomes incoherent

**Comment by superjeng1** [comment]
Ah, ok I understand that now.

**Comment by ragauskl** [comment]
I believe this can be achieved by using Pro version of Super Presets. It is free to set presets per device, but requires Pro to set per app. When you create a new super preset if you enable 'Auto Switching' you'll have the option to which apps this should apply, set all settings to default for that preset and whenever the app in selected list will be activated eqMac will switch to the default/blank preset you've created

<img width="391" alt="image" src="https://github.com/bitgapp/eqMac/assets/9280160/b03567c4-2600-43d3-9513-64003a19ca75">
